javascript/ql/test/query-tests/Performance/ReDoS/tst.js

```diff
@@ -370,4 +370,8 @@ var good43 = /("[^"]*?"|[^"\s]+)+(?=\s*|\s*$)/g;
 
 // BAD
 var bad87 = /("[^"]*?"|[^"\s]+)+(?=\s*|\s*$)X/g;
-var bad88 = /("[^"]*?"|[^"\s]+)+(?=X)/g;
\ No newline at end of file
+var bad88 = /("[^"]*?"|[^"\s]+)+(?=X)/g;
+var bad89 = /(x*)+(?=$)/
+var bad90 = /(x*)+(?=$|y)/
+var bad91 = /([\s\S]*)+(?=$)/
+var bad92 = /([\s\S]*)+(?=$|y)/
```

Comment:
I believe the last two ones should be marked as good? I'm not asking for the query to handle them, just that the test is accurate.

Reply:
You're right. I was a little fast there.